lib/mssql.js

```diff
@@ -365,9 +364,12 @@ MsSQL.prototype.update = MsSQL.prototype.updateAll = function (model, where, dat
   sql = "UPDATE " + tblName + MsSQL.newline;
   sql += "SET " + combined.join() + MsSQL.newline;
   sql += self.buildWhere(model, where) + MsSQL.newline;
+  sql += ';SELECT @@ROWCOUNT as count';
   fieldValues.push(id);
 
-  self.query(sql, fieldValues, cb);
+  self.query(sql, fieldValues, function(err, info) {
+    cb(err, info[0]);
```

Comment:
Is this assuming that `info[0]` is an object with a `count` property? I find it a bit cryptic, can you perhaps add a comment to explain?